libdnf/rpm/solv_query.cpp

```diff
@@ -1466,6 +1466,34 @@ void SolvQuery::Impl::filter_nevra(
                     filter_result.add_unsafe(candidate_id);
                 }
             } break;
+            case libdnf::sack::QueryCmp::IGLOB: {
```

Comment:
libdnf::sack::QueryCmp::IGLOB: could be merged with GLOB, only `if (!all_names && fnmatch(name_c_pattern, candidate_name, 0) != 0) {` could be modify with proper flag. What do you think?

Reply:
I tried to created an optimal solution.
The second solution is shorter, but it must define the fnmatch flag, and it uses one extra comparison. On the other hand, this can be done done before the main loop, so the negative effect is negligible. I will change the PR as you want.